shared/test/loop_mocha_utils.js

```diff
@@ -201,7 +207,7 @@ var LoopMochaUtils = (function(global, _) {
    *                             to the listeners.
    */
   function publish() {
-    var args = Array.slice(arguments);
+    var args = Array.prototype.slice.call(arguments);
```

Comment:
This is the "proper" way to call it:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#Array-like_objects

The other one works in FF, but I don't think its as official as this one.